test/controllers/FeedbackFormsControllerSpec.scala

```diff
@@ -28,7 +28,7 @@ class FeedbackFormsControllerSpec extends PlaySpecification with TestEnvironment
     active = true, BSONObjectID.parse("5943cdd60900000900409b26").get)
```

Comment:
Would you want to change youtubeLink and slideShareLink to youtubeURL and slideShareURL in test cases as well?